docs/api-docs/store-management/shipping/shipping-overview.md

```diff
@@ -1,52 +1,52 @@
+The [Order Shipping Addresses](https://developer.bigcommerce.com/api-reference/orders/orders-api/order-shipping-addresses) endpoint allows you to view the shipping addresses on an order. An order can have more than one shipping address.
 
-## Shipping Zone and Shipping Methods
+## Shipping zone and shipping methods
 
-[Shipping zones ](https://developer.bigcommerce.com/api-reference/store-management/shipping-api/shipping-zones)can be configured in the control panel or using the API to designate where a merchant ships their products and which [shipping methods](https://developer.bigcommerce.com/api-reference/store-management/shipping-api/shipping-method) should be available in each geographic region.
+You can configure [Shipping zones ](https://developer.bigcommerce.com/api-reference/store-management/shipping-api/shipping-zones) in the control panel or use the API to designate where a merchant ships their products and which [shipping methods](https://developer.bigcommerce.com/api-reference/store-management/shipping-api/shipping-method) should be available in each geographic region.
 
 An example of a shipping zone might be North America, and within that shipping zone, the shipping methods might be Free Shipping, Ship-by-Weight, and USPS.
 
-## Real-Time Carriers
+## Real-time carriers
 
-BigCommerce offers native integrations with many real-time shipping carriers such as UPS, Fedex, and USPS. The [Real-Time Carriers](https://developer.bigcommerce.com/api-reference/store-management/shipping-api/shipping-carrier) resource allows you to connect an account for one of our natively supported shipping carrier integrations.
+BigCommerce offers native integrations with many real-time shipping carriers such as UPS, FedEx, and USPS. The [Real-Time Carriers](https://developer.bigcommerce.com/api-reference/store-management/shipping-api/shipping-carrier) resource allows you to connect an account for one of our natively supported shipping carrier integrations.
 
 For a list of all supported carriers, see [Real-Time Quote Providers](https://support.bigcommerce.com/articles/Public/Setting-Up-a-Real-Time-Shipping-Quote-Shipping-Method/#providers).
 
 ## Resources
 
-### Related Endpoints
+### Related endpoints
 * [Storefront Checkout](https://developer.bigcommerce.com/api-reference/cart-checkout/storefront-checkout-api/checkout-consignments/checkoutsconsignmentsbycheckoutidpost)
 * [Server to Server Checkout](https://developer.bigcommerce.com/api-reference/cart-checkout/server-server-checkout-api/checkout-consignments/checkoutsconsignmentsbycheckoutidpost)
 * [Order Shipping Addresses](https://developer.bigcommerce.com/api-reference/orders/orders-api/order-shipping-addresses)
 * [Shipping Methods](https://developer.bigcommerce.com/api-reference/store-management/shipping-api/shipping-method/createashippingmethod)

```

Comment:
I suggest replacing Shipping Methods, Shipping Zones, and Real-Time Carriers endpoints with a link to the [Shipping API](https://developer.bigcommerce.com/api-reference/store-management/shipping-api). The linked document contains reference to the aforementioned endpoints.

Reply:
Yes, that sounds good. I will update the link.